### PR TITLE
appliance/postgres: Add DATABASE_URL to postgres provider

### LINKED ISSUE #2020: Automatically set DATABASE_URL when Postgres resource is used
To work out of the box with more Heroku-compatible apps, we can set `DATABASE_URL` automatically when the PG resource is added.

Currently we recommend this command:

    flynn env set DATABASE_URL=$(export $(flynn env | xargs); echo "postgres://$PGUSER:$PGPASSWORD@$PGHOST:5432/$PGDATABASE")

### PR DESCRIPTION
**Pull request by josephglanville**
Closes #2020

### CODE REVIEW

**Comment by josephglanville** [comment]
Docs will probably need updating to reflect that you no longer need to set `DATABASE_URL` manually.

**Comment by titanous** [comment]
docs are here: https://github.com/flynn/flynn/blob/master/docs/content/postgres.html.md

**Comment by josephglanville** [comment]
Cool, I will fix up the docs too. :)

File: docs/content/postgres.html.md [hunk 1]
```diff
@@ -29,14 +29,7 @@ Provisioning the database will add a few environment variables to your app
...
 
-#### DATABASE_URL
-
-If your application expects a `DATABASE_URL` environment variable, you can
-create one by combining the `PG*` variables:
-
-```text
-flynn env set DATABASE_URL=$(export $(flynn env | xargs); echo "postgres://$PGUSER:$PGPASSWORD@$PGHOST:5432/$PGDATABASE")
-```
+Flynn will also create the `DATABASE_URL` environment which is utilized by some frameworks to configure database connections.
```
Comment (titanous) [inline docs/content/postgres.html.md]: s/environment/environment variable/

**Comment by titanous** [comment]
LGTM

**Comment by kwando** [comment]
:heart:

**Pull request merged**